CMakeLists.txt

```diff
@@ -792,6 +793,10 @@ if(WITH_OPTIM)
                 set(WITH_SSE4 OFF)
             endif()
         endif()
+        if(FORCE_TZCNT)
```

Comment:
It might be better if this block was near FORCE_SSE2 in CMake..

Reply:
Actually, it looks like in the CMakeLists there is a `cmake_dependent_option`. 
![image](https://user-images.githubusercontent.com/1364432/158231639-cc7102d8-9e5a-4cee-bab8-0e63233bea5c.png)

So that might be different and perhaps it is okay where it is.